test/child/ChildChainManager.test.js

```diff
@@ -186,7 +186,7 @@ contract('ChildChainManager', async(accounts) => {
     })
 
     it('Token should not exist before deposit', async() => {
-      await expectRevert(contracts.child.dummyERC721.ownerOf(depositTokenId), 'ERC721: owner query for nonexistent token')
+      await expectRevert(contracts.child.dummyERC721.ownerOf(depositTokenId), 'value out of range')
```

Comment:
This is regarding -- https://forum.openzeppelin.com/t/require-in-view-pure-functions-dont-revert-on-public-networks/1211

Reply:
So we should expect these tests to fail locally? And how did the tests pass on CI in previous PRs?

Reply:
@arthcp It was running on ganache instead of Bor.